cmds/exp/cpu/cpu.go

```diff
@@ -43,16 +46,33 @@ var (
 	remote    = flag.Bool("remote", false, "indicates we are the remote side of the cpu session")
 	network   = flag.String("network", "tcp", "network to use")
 	keyFile   = flag.String("key", filepath.Join(os.Getenv("HOME"), ".ssh/cpu_rsa"), "key file")
-	srv9p     = flag.String("srv", "unpfs", "what server to run")
 	bin       = flag.String("bin", "cpu", "path of cpu binary")
 	port9p    = flag.String("port9p", "", "port9p # on remote machine for 9p mount")
 	dbg9p     = flag.Bool("dbg9p", false, "show 9p io")
 	root      = flag.String("root", "/", "9p root")
-	bindover  = flag.String("bindover", "/lib:/lib64:/lib32:/usr:/bin:/etc", ": separated list of directories in /tmp/cpu to bind over /")
+	bindover  = flag.String("bindover", "/lib:/lib64:/lib32:/usr:/bin:/etc:/home", ": separated list of directories in /tmp/cpu to bind over /")
+	mountopts = flag.String("mountopts", "", "Extra options to add to the 9p mount")
+	msize     = flag.Int("msize", 1048576, "msize to use")
 )
 
 func verbose(f string, a ...interface{}) {
-	v(f+"\r\n", a...)
+	v("\r\n"+f+"\r\n", a...)
+}
+
+// getNonce returns a nonce, or an error if random reader fails.
+func getNonce() (nonce, error) {
```

Comment:
Perhaps a better name for this function would be `generateNonce`. `get` kind of implies the nonce is coming from the other machine.

Reply:
done